openfisca_us/tests/policy/baseline/gov/states/in/tax/income/taxes/in_county_tax.yaml

```
@@ -0,0 +1,20 @@
+- name: IN county tax
+  period: 2021
+  input:
+    people:
```

We reserve `variable_name.yaml` for unit testing `variable_name`, so the only inputs are the inputs to the function itself. More involved tests would go in an `integration.yaml` in the same folder if you'd like to add that. Though in this case I think the unit test suffices.
```suggestion
    in_agi: 1_000
    county_str: MONROE_COUNTY_IN
```